libs/ledger-live-common/src/families/vechain/bridge.integration.test.ts

```diff
@@ -0,0 +1,212 @@
+import "../../__tests__/test-helpers/setup";
+import { testBridge } from "../../__tests__/test-helpers/bridge";
+import type { AccountRaw, DatasetTest } from "@ledgerhq/types-live";
+import type { Transaction } from "./types";
+import { fromTransactionRaw } from "./transaction";
+import { DEFAULT_GAS_COEFFICIENT, MAINNET_CHAIN_TAG } from "./constants";
+import { vechain1 } from "./datasets/vechain";
+import { generateNonce } from "./utils/transaction-utils";
+import BigNumber from "bignumber.js";
+import {
+  setSupportedCurrencies,
+  listSupportedCurrencies,
+  getCryptoCurrencyById,
+} from "../../currencies";
+import vechainScanAccounts1 from "./datasets/vechain.scanAccounts.1";
+import { AmountRequired, NotEnoughBalance } from "@ledgerhq/errors";
+import VIP180 from "./contracts/abis/VIP180";
+import { CryptoCurrencyId } from "@ledgerhq/types-cryptoassets";
+
+const listSupported = listSupportedCurrencies();
+listSupported.push(getCryptoCurrencyById("vechain"));
+setSupportedCurrencies(listSupported.map(c => c.id) as CryptoCurrencyId[]);
+
+const dataset: DatasetTest<Transaction> = {
+  implementations: ["js"],
+  currencies: {
+    vechain: {
+      FIXME_ignoreAccountFields: ["balance", "spendableBalance", "estimateMaxSpendable"],
+      scanAccounts: vechainScanAccounts1,
+      accounts: [
+        {
+          raw: vechain1 as AccountRaw,
+          implementations: ["js"],
+          transactions: [
```

Comment:
I do not have strong knowledge of bridge tests but the NotEnoughVTHO depends on the balance of the account, 
Since the average fees are about 0.21 vtho, it would mean that we need to have an account with some vet and 0 vtho, can we reach that, with an isolated test, replicating this every time?

Reply:
added NotEnoughVTHO case 👍